src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -151,6 +151,33 @@ protected SemanticModel GetSemanticModel(SyntaxTree tree)
 
                 AddOverloadCountPart(symbols);
                 FixAllAnonymousTypes(symbols[0]);
+                AddExceptions(symbols[0]);
+            }
+
+            private void AddExceptions(ISymbol symbol)
+            {
+                // clean up the list of possible exceptions by de-duplicating and ordering them
+                var exceptions =
+                    symbol.GetDocumentationComment().ExceptionTypes
+                    .Distinct()
+                    .OrderBy(e => e)
```

Comment:
Then this brings up the issue of testability again.  If we use the current culture then we could presumably get a different sort order when running unit tests on other languages and that's a maintenance nightmare (coming from the person who had to make all of these tests run on various cultures a few months ago.)

A question for @Pilchie and/or @DustinCampbell: what comparer should I specify for the sort, or should we fall back to not sorting anything and displaying the exceptions in the order in which they were declared in the original doc comment?